Extensions/PlatformBehavior/platformerobjectruntimebehavior.js

```diff
@@ -933,6 +933,46 @@ gdjs.PlatformerObjectRuntimeBehavior.prototype.getJumpSustainTime = function() {
   return this._jumpSustainTime;
 };
 
+/**
+ * Get the current speed during a fall of the Platformer Object.
```

Comment:
Get the speed at which the object is falling. It is 0 when the object is on a floor, and non 0 as soon as the object leaves the floor.